user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/IGQIRowOperator.md

```diff
@@ -0,0 +1,22 @@
+---
+uid: GQI_IGQIRowOperator
+---
+
+# IGQIRowOperator Interface
+
+## Definition
+
+Namespace: `Skyline.DataMiner.Analytics.GenericInterface`  
+Assembly: `SLAnalyticsTypes.dll`
+
+This interface makes it possible to manipulate the rows in a custom operator.
+
+## Methods
+
+### void HandleRow(GQIEditableRow row)
+
+Makes it possible to manipulate the rows the custom operator is working with.
```

Comment:
I can imagine this description isn't very helpful to implementers
It would be better to say:

1. when this method is called
2. in broad terms, what you could/should do in its implementation

Something like:
This method is called exactly once for every row when executing a query.
It allows you to get or set cell values from the given row.